src/elements/content-sidebar/activity-feed/comment/components/stories/BaseCommentMenu.stories.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+import * as React from 'react';
```

Comment:
We may want to wait to start down this path, especially since this change is going to be quite large due the refactor. Do we need stories for low-level components that are scoped to a single "experience"?

Reply:
The main reason I created these stories---aside from just liking Storybook development---is because Preview kinks (like monoproxy) had not been worked out for dev pods. During planning, we were going to suspend working on these tickets, but I suggested we could go ahead using Storybook. I think a trail has since been blazed to use dev pods for Preview dev, and I plan to pick up that trail to better confirm these changes before I move this PR out of draft. But, in sum, the stories were the means by which I developed these components.